app/components/m-header.js

```diff
@@ -1,46 +1,53 @@
-import Component from '@ember/component';
+
 import ENV from 'frontend-kaleidos/config/environment';
 import { inject as service } from '@ember/service';
-import { computed } from '@ember/object';
+import {
+  computed, action
+} from '@ember/object';
 import { isEmpty } from '@ember/utils';
+import { tracked } from '@glimmer/tracking';
+import Component from '@glimmer/component';
+
 
-export default Component.extend({
-  currentSession: service(),
-  classNames: ['auk-u-block'],
+export default class MHeader extends Component {
+  @service currentSession;
+  @service router;
+
+  @tracked hasPublicationsEnabled = !isEmpty(ENV.APP.ENABLE_PUBLICATIONS_TAB);
```

Comment:
Native get function ipv `@tracked` lijkt me geschikter omdat deze waarde eerder static is.